src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php

```diff
@@ -46,23 +44,33 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $source = $input->getArgument('source');
-
-        $bundlePath = $this->getApplication()->getKernel()->getBundle('MaxmindGeoipBundle')->getPath();
+        /** @var \Symfony\Bundle\FrameworkBundle\Console\Application $application */
```

Comment:
`$this->getApplication()` returns Symfony\Component\Console\Application which does not have method getKernel. With this annotation you'll see `getKernel()` method.